packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -96,18 +96,25 @@ export const createDefaultCallingHandlers: CreateDefaultCallingHandlers = memoiz
   };
 });
 
-/* @conditional-compile-remove(spotlight) */
+/* @conditional-compile-remove(spotlight) */ /* @conditional-compile-remove(soft-mute) */
 /**
  * Handlers only for calling components
  * @internal
  */
 export interface _ComponentCallingHandlers {
+  /* @conditional-compile-remove(spotlight) */
   /** VideoGallery callback prop to start local spotlight */
   onStartLocalSpotlight: () => Promise<void>;
+  /* @conditional-compile-remove(spotlight) */
   /** VideoGallery callback prop to stop local spotlight */
   onStopLocalSpotlight: () => Promise<void>;
+  /* @conditional-compile-remove(spotlight) */
   /** VideoGallery callback prop to start remote spotlight */
   onStartRemoteSpotlight: (userIds: string[]) => Promise<void>;
+  /* @conditional-compile-remove(spotlight) */
   /** VideoGallery callback prop to stop remote spotlight */
   onStopRemoteSpotlight: (userIds: string[]) => Promise<void>;
+  /* @conditional-compile-remove(soft-mute) */
+  /** VideoGallery callback prop to mute remote participant */
+  onMuteParticipant: (userId: string) => Promise<void>;
```

Comment:
Could we not put `onMuteParticipant` in CommonCallingHandlers so that `onMuteParticipant` is also an adapter function? Or did you intend it to not be an adapter function like the spotlight functions here?

Reply:
Great point! Removed as it wasn't needed